webui/Makefile

```diff
@@ -2,20 +2,23 @@
 
 default: get-deps build
 
-clean:
-	$(MAKE) -C react $@
-	$(MAKE) -C elm $@
-	$(MAKE) -C tests $@
-
-fmt:
-	$(MAKE) -C react $@
-	$(MAKE) -C elm $@
-	$(MAKE) -C tests $@
-
-check:
-	$(MAKE) -C react $@
-	$(MAKE) -C elm $@
-	$(MAKE) -C tests $@
+clean: clean-react clean-elm clean-tests
+	$(MAKE) -C api-ts-sdk $@
+.PHONY: clean-%
+clean-%:
+	$(MAKE) -C $(subst -,/,$*) clean
+
+.PHONY: fmt
+fmt: fmt-react fmt-elm fmt-tests
+.PHONY: fmt-%
+fmt-%:
+	$(MAKE) -C $(subst -,/,$*) clean
+
+.PHONY: check
+check: check-react check-elm check-tests
+.PHONY: check-%
+check-%:
+	$(MAKE) -C $(subst -,/,$*) clean
```

Comment:
**question:** what is this black magic?

Reply:
haha it is the same excluding the added clean line for `api-ts-sdk` in the clean target. 
this is to make it look more like the project's main Makefile.